### ISSUE TITLE
Enable UK shipments

### ISSUE THREAD

**Issue by joolaay** [DoSomethingArchive/legacy-website#4732]
![screen shot 2015-07-23 at 3 29 54 pm](https://cloud.githubusercontent.com/assets/6331017/8860319/ff9ac1fe-3150-11e5-8aaa-8ebaee16b09c.png)

The UK team is running a version of 1 in 3 of us and needs to collect addresses in order to mail out their bracelets. 
We need to 1) Enable the capability in custom settings to set up a shipment form. and 2) Adjust it for the UK: Instead of Zip Code, have it labeled "Post Code." State should be "County" and City should be "Town/City". 

We won't be adding validation. 

Campaign launches Aug 12.

**Comment by angaither** [comment]
@deadlybutter we need to make sure the form fields change for uk only, as well as ensure that the US validation still works, but doesn't try to fire on uk forms

**Comment by joolaay** [comment]
@deadlybutter UK is looking for a timing update - anything I can tell them for now?

**Comment by itsjoekent** [comment]
@julielorch was only able to start working on this one today, had issues getting my international environment running but sergii helped me out. Hopefully I knock this out by EOD!

**Comment by joolaay** [comment]
Woooord!

**Comment by itsjoekent** [comment]
this is the current form that already exists when i enable shipments on the UK site, looks like it just needs county added.

<img width="576" alt="screen shot 2015-07-30 at 1 58 02 pm" src="https://cloud.githubusercontent.com/assets/897368/8990885/19ee3c9e-36c3-11e5-8c08-524ee660d999.png">

**Comment by itsjoekent** [comment]
actually @julielorch does the order here matter?

**Comment by joolaay** [comment]
My hunch is that yes, the order matters. It should reflect the first pasted
image. I think the UK guys are signed off for now, but I could ask them
tomorrow if it matters.

**Comment by sergiitk** [comment]
Hi @julielorch, @mikefantini.
I enabled the form on local UK and got shipment form just like @deadlybutter's.
I looked into technical approaches for two adjustments and concluded:
1. Enable country field.
   As I understand, these field intended for [UK countries](https://en.wikipedia.org/wiki/United_Kingdom):  
   
   > The UK consists of four countries: England, Scotland, Wales, and Northern Ireland.  
   
   However, Drupal doesn't support these, only ISO standardized country names are allowed: https://github.com/drupal/drupal/blob/7.x/includes/iso.inc#L103
   Hence it would be impossible to save the form with Wales or Scotland in country field.
2. Fields order. Now, when we can't use country field it doesn't make much sense. Plus, it turned out to be not so easy as we expected: it's impossible to use normal `#weight` attribute to change order of address fields (discussed with @angaither, source: http://drupal.stackexchange.com/questions/52047/rearranging-subfields-with-the-address-field-module)

Except country field and field order, everything looks like good to go.
